### PR TITLE
Activate deactivate signal

### PR DESCRIPTION
**Pull request by tspenov**
Activate/deactivate trigger:

```graphql
     mutation {
        updateTrigger(
          id: 123,
          active: false
        ) {
          trigger{
            active
          }
        }
      }
```

Trigger is `repeating` by default. If not `repeating` then it is triggered once and deactivated.

### CODE REVIEW

File: lib/sanbase/signals/evaluator/scheduler.ex [hunk 1]
```diff
@@ -81,6 +84,17 @@ defmodule Sanbase.Signals.Scheduler do
...
 
+  defp deactivate_non_repeating(triggers) do
+    triggers
+    |> Enum.filter(fn ut -> !ut.trigger.repeating end)
+    |> Enum.map(fn %UserTrigger{id: id, user: user} ->
+      UserTrigger.update_user_trigger(user, %{
+        id: id,
+        active: false
+      })
+    end)
```
Comment (IvanIvanoff) [inline lib/sanbase/signals/evaluator/scheduler.ex]: You can do this in a single pass with `for` like this:
```elixir
for %UserTrigger{id: id, trigger: %Trigger{repeating: false}} <- triggers do
  UserTrigger.update_user_trigger(user, %{
    id: id,
    active: false
  })
end
```

Comment (tspenov) [reply lib/sanbase/signals/evaluator/scheduler.ex]: nice

File: lib/sanbase/signals/trigger/settings/price_absolute_change_settings.ex [hunk 2]
```diff
@@ -105,7 +102,7 @@ defmodule Sanbase.Signals.Trigger.PriceAbsoluteChangeSettings do
 
     @doc ~s"""
     Construct a cache key only out of the parameters that determine the outcome.
-    Parameters like `repeating` and `channel` are discarded. The `type` is included
+    Parameters like `channel` is discarded. The `type` is included
```
Comment (IvanIvanoff) [inline lib/sanbase/signals/trigger/settings/price_absolute_change_settings.ex]: `Parameters ... are`

File: lib/sanbase/signals/trigger/settings/price_percent_change_settings.ex [hunk 3]
```diff
@@ -123,7 +120,7 @@ defmodule Sanbase.Signals.Trigger.PricePercentChangeSettings do
 
     @doc ~s"""
     Construct a cache key only out of the parameters that determine the outcome.
-    Parameters like `repeating` and `channel` are discarded. The `type` is included
+    Parameters like `channel` is discarded. The `type` is included
```
Comment (IvanIvanoff) [inline lib/sanbase/signals/trigger/settings/price_percent_change_settings.ex]: `Parameters ... are`

File: lib/sanbase_web/graphql/schema/user_trigger_types.ex [hunk 4]
```diff
@@ -15,5 +15,6 @@ defmodule SanbaseWeb.Graphql.UserTriggerTypes do
     field(:is_public, :boolean)
     field(:cooldown, :string)
     field(:tags, list_of(:tag))
+    field(:active, :boolean)
```
Comment (IvanIvanoff) [inline lib/sanbase_web/graphql/schema/user_trigger_types.ex]: how about adding `non_null`s where possible?

**Pull request merged**